docstring_parser/parser/__init__.py

```diff
@@ -5,14 +5,23 @@
 from . import google, rest
 from .common import Docstring, ParseError
 
+try:
+    from . import numpy
+except ImportError:
+    numpy = None
+
 
 class Style(enum.Enum):
     rest = enum.auto()
     google = enum.auto()
     auto = enum.auto()
+    if numpy:
+        numpy = enum.auto()
```

Comment:
I don't like this, the user will get `AttributeError` when they try to use `Style.numpy` without having `numpy` installed. I'd expect some more informative error when they run the `parse` function.

Reply:
In my other libraries I raise the `ImportError` that I catch above whenever a user tries to use the `numpy` functionality. I will implement this generically